packages/browser/src/login/oidc/redirectHandler/ErrorOidcHandler.ts

```diff
@@ -51,20 +53,14 @@ export class ErrorOidcHandler implements IRedirectHandler {
 
   async handle(
     redirectUrl: string,
-    // The argument is ignored, but must be present to implement the interface
-    _onToken?: (newToken: string) => unknown,
-    onError?: (
-      error: string | null,
-      errorDescription?: string | null
-    ) => unknown
+    eventEmitter?: EventEmitter
   ): Promise<ISessionInfo & { fetch: typeof fetch }> {
-    if (typeof onError === "function") {
+    if (eventEmitter) {
```

Comment:
```suggestion
    if (eventEmitter !== undefined) {
```
This won't change the behaviour here, but it's technically a stricter test, given we know that `eventEmitter` is either an object of type `EventEmitter` or `undefined`, and not some "falseish" value like an empty string for instance.